l10n_fi_edicode/models/res_partner_operator_einvoice.py

```diff
@@ -0,0 +1,62 @@
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class ResPartnerOperatorEinvoice(models.Model):
+    _name = "res.partner.operator.einvoice"
+    _description = "eInvoice Operator"
+    _order = "sequence, id"
+
+    name = fields.Char(string="Operator", required=True)
+    active = fields.Boolean(default=True)
+    sequence = fields.Integer()
+    identifier = fields.Char(
+        required=True,
+        size=35,
+        help="Monetary Institution Identifier (see https://tieke.fi)",
+    )
+    ttype = fields.Selection(
+        [
+            ("bank", "Bank with Finvoice brokerage service"),
+            ("broker", "Carrier broker"),  # default
+        ],
+        "Type",
+        default="broker",
+    )
+
+    _sql_constraints = [
+        (
+            "operator_identifier_uniq",
+            "unique(identifier)",
+            '"Identifier" should be unique!',
+        ),
+    ]
+
+    @api.depends("identifier")
+    def _compute_display_name(self):
+        """
+        Overwrite core method to add value of `identifier` ("Identifier") field
+        into name of records.
+        """
+        res = super()._compute_display_name()
+        for operator in self:
+            operator.display_name = " - ".join([operator.identifier, operator.name])
+        return res
```

Comment:
What's the purpose of returning `res` in a compute method?

Reply:
it is a pylint check (W8110) that makes sure that you do not break the result of calling 'super'. Obviously in the compute methods you should not expect any result, but it is done for convenience and good practice

https://github.com/OCA/pylint-odoo/blob/6e857c52117ef8597e0433b32ca6fac302ea76b8/src/pylint_odoo/checkers/odoo_addons.py#L208

Reply:
Welp, I don't see how it could cause any harm.